provider/spark.go

```diff
@@ -1829,6 +1868,9 @@ func blobSparkMaterialization(id ResourceID, spark *SparkOfflineStore, isUpdate
 		spark.Logger.Errorw("Problem creating spark submit arguments", "error", err)
 		return nil, fmt.Errorf("error with getting spark submit arguments %v", sparkArgs)
 	}
+	if outputFormat == filestore.CSV {
```

Comment:
can we just always append the output format regardless of it being CSV or not?

Reply:
So, I _can_ do this, but I set the default value for this arg to parquet. Given this, do you still want to append parquet to all arg lists or simple add a comment noting this?